infrastructure/server-setup/tasks/updates.yml

```diff
@@ -21,8 +21,31 @@
       Unattended-Upgrade::Remove-New-Unused-Dependencies "true";
       Unattended-Upgrade::Remove-Unused-Dependencies "true";
       Unattended-Upgrade::Allowed-Origins {
+          "${distro_id}:${distro_codename}";
           "${distro_id}:${distro_codename}-security";
+          "${distro_id}ESMApps:${distro_codename}-apps-security";
+          "${distro_id}ESM:${distro_codename}-infra-security";
+          "${distro_id}:${distro_codename}-updates";
       };
+
     owner: root
     group: root
     mode: '0644'
+
+- name: Autoclean Unattended Upgrades    
+  ansible.builtin.copy:
+    dest: /etc/apt/apt.conf.d/20auto-upgrades
+    content: |
+      APT::Periodic::Update-Package-Lists "1";
+      APT::Periodic::Unattended-Upgrade "1";
+      APT::Periodic::AutocleanInterval "7";
+
+    owner: root
+    group: root
+    mode: '0644'
+
+- name: Restart Unattended-upgrades service
```

Comment:
My ansible is not that great but as I understand this will restart this service on every ansible run?

Is restart needed for the the config changes? Should we hook it in to detecting state change in the configs and then restarting (or ideally reloading)?

Reply:
As it is config changes there is a restart required to be done. As per my understanding reload will only help to start a service if it is stopped.